borg/archive.py

```diff
@@ -9,6 +9,7 @@
 logger = create_logger()
```

Comment:
It is a bit unfortunate that there was no discussion about translation before this changeset, because you invested quite some work in it and I think I'ld prefer not having any translation.

translations are a burden and they tend to be outdated once the initial effort / passion is over. I know this from other projects, translations are always behind. sometimes, translations are even worse than just having stuff in english (that's why I have my system set to en-US to not having to bear with crappy german translations).

Also, how many other UNIX commandline system tools do you know which speak something else than english to you?

Where would translation start, where stop? Long commandline options are english, too. Exception messages? Do I have to speak french if I want to fully understand a traceback from a french locale using user? Do we need translated docs also, because the input/output of the program would be different?

Sorry, but currently the effort needed to implement and maintain this seems to by far outweigh any positive effect this might have on some users.